### PR TITLE
Section 2 - Paul Rene Nichols

### PR DESCRIPTION
**Pull request by paulrenenichols**
- wrote lesson zero test

### CODE REVIEW

**Comment by gizemny** [comment]
Great job!

File: test/00LessonZeroTest.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-    return false;
+    return true;
 }
```
Comment (paulrenenichols) [inline test/00LessonZeroTest.js:8]: not sure this works, @gizemny. Halp!!!!

File: test/01LessonOneTest.js [hunk 2]
```diff
@@ -5,11 +5,12 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+    return true;
 }
 
 function returnFalse() {
     // should return false
+    return false;
```
Comment (jasonburks) [inline test/01LessonOneTest.js]: What were you thinking!

File: test/01LessonOneTest.js [hunk 3]
```diff
@@ -5,11 +5,12 @@ var assert = require('assert');
...
 
 }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:15]: Guess what Paul, you caint spell!!!
- one
- two

[link text](https://media.giphy.com/media/j2rCxv6luaLq8/giphy.gif)

link text

File: test/01LessonOneTest.js (hunk 2 above)
Comment (andrewcfisher) [inline test/01LessonOneTest.js]: Amazing. Really amazing.

Comment (paulrenenichols) [inline test/01LessonOneTest.js]: @jasonburks What what!

**Pull request closed without merging**